src/templates/fields/radio.eco

```diff
@@ -1 +1,27 @@
-<%- JST["partials/options_field"](@) %>
+<% for option in @model.getOptions(): %>
```

Comment:
this has been changed from options_field.eco to use the new `dobtradiogroup` binder.